lca-cli/postpivot/postpivot.go

```diff
@@ -591,3 +601,112 @@ func (p *PostPivot) createPullSecretManifest(pullSecret, pullSecretManifest stri
 	}
 	return nil
 }
+
+// waifForConfiguration waits till configuration folder exists or till device with label "relocation-config" is added.
+// in case device was provided we are mounting it and copying files to configFolder
+func (p *PostPivot) waifForConfiguration(ctx context.Context, configFolder, blockDeviceMountFolder string) error {
+	err := wait.PollUntilContextCancel(ctx, time.Second, true, func(ctx context.Context) (bool, error) {
```

Comment:
What happens in case of error?

Reply:
Well, mount may fail, why not retry?

Reply:
We can, i thought that it sort of error that you will just always fail no?